src/jobs/ega/axios/egaClient.ts

```diff
@@ -146,7 +84,13 @@ export const egaApiClient = async () => {
 
   const getAccessToken = async (): Promise<IdpToken> => {
     if (currentToken) {
-      return currentToken;
+      const tokenIsExpired = await tokenExpired(currentToken);
+      if (tokenIsExpired) {
```

Comment:
I'm looking at where this tokenExpiry check is called, and I'm not sure if it is useful? since the only places `getAccessToken` is called is on the initial EgaClient creation, and after a 401 occurs.